### DISCUSSION TITLE
games from date

### DISCUSSION THREAD

**Discussion by axiles89** [kompot/nhl-tv-geeky-streams#58] [Q&A]
How can I specify the exact date to display the list of games? pushing back a month is a bad idea

**Comment by rseanhall** [comment]
`docker-compose run --rm nhltv yarn start --start-date 2022-05-19` will show games from May 19th.

`docker-compose run --rm nhltv yarn start --help` will show all command line options.

**Comment by axiles89** [comment]
thanks!

**Reply by rseanhall** [reply]
Were you able to get it to work? I saw an email notification about a different problem but it seems to be deleted.
